include/revng/Model/NamingConfiguration.h

```diff
@@ -0,0 +1,223 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+/* TUPLE-TREE-YAML
+name: NamingConfiguration
+type: struct
+fields:
+  - name: UnnamedSegmentPrefix
+    doc: |
+      The prefix for a segment without a name.
+
+      The default value is `segment_`.
+    type: string
+    optional: true
+
+  - name: UnnamedFunctionPrefix
+    doc: |
+      The prefix for a local function without a name.
```

Comment:
Indentation

Reply:
Wait, what do you mean? These are not checked by the check-conventions?!

Reply:
No, these are comments. I think we run prettify on `.yml` files, but no in the middle of comments.

Reply:
These should really become their own files sooner rather that later :cursed_parrot:

Reply:
That being said, I just copied what was there before: https://github.com/revng/revng/blob/develop/include/revng/Model/ABI.h#L15

How do you see us formatting these?

Reply:
I'm fine with copying things over, but here the indentation is different from the next line.

Reply:
Ah, fuck, a typo. Sorry for the noise and a good catch!

Reply:
Fixed.